src/test/java/co/rsk/federate/FederatorSupportTest.java

```diff
@@ -362,6 +358,107 @@ void getProposedFederationCreationBlockNumber_whenBlockNumberIsValid_shouldRetur
         assertEquals(expectedBlockNumber.longValue(), result.get());
     }
 
+    @Test
+    void getActiveFederationCreationTime_whenCreationTimeIsValid_preRSKIP419_shouldReturnInstantFromMillis() {
+        // Arrange
+        BigInteger expectedCreationTime = BigInteger.valueOf(System.currentTimeMillis());
+        when(bridgeTransactionSender.callTx(any(), eq(Bridge.GET_FEDERATION_CREATION_TIME)))
+            .thenReturn(expectedCreationTime);
+
+        // all rskips are activated since block 0
+        org.ethereum.core.Block mockBlock = mock(org.ethereum.core.Block.class);
+        when(mockBlock.getNumber()).thenReturn(-1L);
```

Comment:
didnt find a better way to do this, since the test environment has all rskips activated